test/fixtures/resolver/ts/import-syntax/main.ts

```diff
@@ -16,3 +16,7 @@ import "module_package_dir";
 import $ from "jquery";
 import i = require("./Import");
 import Array from "./Array";
+
+export * from "./Alias";
+export * from "./DoubleQuote";
+export * from "./Simple";
```

Comment:
Maybe also add some cases to make sure they do not match the other cases. (see https://www.typescriptlang.org/docs/handbook/modules.html)

```
export { ZipCodeValidator as mainValidator };
export default "123";
export = ZipCodeValidator;
export var t = something + 1;
```